membership-baseline.md

```diff
@@ -0,0 +1,169 @@
+    - One month free pluralsight membership
+    - Free access to security training programs by - <Partner with learning platforms/companies and ask for free training subscriptions>
+- <Add more benefits here> related to learning resources
+
+**Note:** We can not promise the work in progress benefits as it mainly depends on the vendor and their community partnership policy.
+
+# Networking Opportunities:
+
+- Engage with like-minded professionals through discussion groups such as Discord, Whatsapp and social media handles
+- Attend chapter wise security community meetups, workshops and various other events. More details at https://null.community/chapters.
+
+
+# Discounts and Offers:
+
+- Being part of null gives you an added advantage of getting special priority discounts on global and participating regional security conferences, training programs or any other events with which null has partnered with. Giving you another reason to connect with industry leaders and expand your professional network.
+    - Few previous partnership are mentioned below 
+        - Nullcon
+        - Rootconf
+        - BSides Delhi
+        - Threatcon
+        - BSides Bangalore
+# Individual Membership Benefits
+
+- Voting rights in null Global Board elections
+- Complimentary null.community email address
+- Few other work in progress benefits:
+    - Group insurance: We are currently exploring the possibility of providing group insurance benefits to our community members. Our team is in discussions with insurance providers to offer coverage tailored to the needs of security professionals. Stay tuned for updates on this exciting opportunity!
+
+
+**Note:** We can not promise the work in  progress benefits as it mainly depends on the vendor and their community partnership policy
+
+# Membership Types and Fees:
+Choose the membership type that suits you best:
+
+## Regular Membership:
+
+- Annual fee: $X
+- Full access to all community resources and benefits.
+
+## Student Membership:
+
+- Annual fee: $X (with proof of enrollment)
+- Discounted rate for students with a passion for security.
+
+## Lifetime Membership:
+
+- One-time payment: $X
+- Enjoy lifelong access to all community resources and benefits.
+
+# Membership Process:
+To become a member of null, please follow these steps:
+
+## Step 1: Submitting Membership Application:
+
+- WIP
+    - Ex: Where one can fill the form to become member
+    - What happens next ?
+
+## Step 2: Payment Processing:
+
+- WIP
+    - Where can they make the payment
+    - What types of payment modes are supported by the platform if used any
+
+## Step 3: Membership Record Management:
+
+- WIP
+    - How are we planning to keep record of the payment
+    - How do we plan to keep record of each member
+    - Is the provided solution scalable ?
+
+
+# Rules & Regulations:
+- Membership dues are the responsibility of the member and cannot be subsidized or funded by Null Community projects, chapters, or events.
```

Comment:
that is a conversation to be had after we have finalized on the cost. if the cost is reseaonably low then no point in discounting it. but if its coming out to be big number in indian context then we can think about this. However since we are also talking about international membership. I would also be recommending we do regional pricing so effectively someone becoming a member from USA needs to pay a higher cost them someone from india coz of pay parity.

Reply:
TBD

Reply:
To keep it as easy/simple as possible and think about these later - Discussed in biweekly call